docs/features.md

```diff
@@ -0,0 +1,6 @@
+# Future feats
```

Comment:
Might make sense to record these as Github "issues"

Reply:
Yes